be/src/service/starrocks_main.cpp

```diff
@@ -215,10 +227,19 @@ int main(int argc, char** argv) {
     options.compaction_mem_tracker = exec_env->compaction_mem_tracker();
     options.update_mem_tracker = exec_env->update_mem_tracker();
     starrocks::StorageEngine* engine = nullptr;
-    auto st = starrocks::StorageEngine::open(options, &engine);
-    if (!st.ok()) {
-        LOG(FATAL) << "fail to open StorageEngine, res=" << st.get_error_msg();
-        exit(-1);
+
+    if (as_cn) {
+        auto st = starrocks::EmptyStorageEngine::open(options, string(getenv("STARROCKS_HOME")) + "/conf/", &engine);
```

Comment:
Is there any way to put `conf_path` into storage options? Because that will make code looks clear, because we probably think: to create storage engine, all your need is `StorageEngineOptions`.